mlc_llm/quantization/__init__.py

```diff
@@ -39,18 +63,24 @@
             group_size=32,
             transpose=True,
         ),
-        embedding_table=GroupQuantizationSpec(
+        embedding_table=NoQuantizationSpec("float16"),
+        final_fc_weight=NoQuantizationSpec("float16"),
+    ),
+    "q4f16_1": QuantizationScheme(
+        name="q4f16_1",
+        linear_weight=GroupQuantizationSpec(
```

Comment:
Interesting to see the existing quantization methods got updated. Do we have some particular reason to update the scheme of `q4f16_0` and `q4f16_1` here? Updating the existing scheme is a bit risky as this will break the prebuilt libraries and weights, so it is always more encouraged to add new schemes.

Reply:
nop, just for debugging.. I'm removing some redundant code that was used for debugging purposes.

Reply:
thanks ruihang, these code has been recovered.